PNA.mdk

```diff
@@ -930,7 +930,10 @@ property is `true` for a table, the P4 developer is allowed to define
 a default action for the table that calls the `add_entry` extern
 function.  `add_entry` adds a new entry to the table whose default
 action calls the `add_entry` function.  The new entry will have the
-same key that was just looked up.
+same key that was just looked up. The new entry value, as in a (key,value)
+pair, is obtained from sources such as packet headers that caused the
+table lookup to miss. Other sources are metadata and results from
+prior processing of the P4 program before `add_entry` is called.
```

Comment:
I agree all of this new text is abolutely true, but it seems redundant to me with the definition in the P4 language spec of what the expressions in a table's `key` expressions are allowed to be.  Is it worth listing these possibilities out?

Reply:
A user may be used to configuring table entries only from the control-plane. I was confused. Such a user will scratch his head and ask where does add-on-miss get its value data from to be able to write a key-value to memory.

Reply:
But the place where the new entry gets its parameter values is from the parameters to `add_entry`.  Why not just say that?  Why say "they might be packet header fields, they might be metadata calculated earlier, etc."  The rules of P4 say how to evaluate those expressions, without needing to list them all again in the definition of add_entry.